filters/f_decoder_wrapper.c

```diff
@@ -113,7 +113,7 @@ static int decoder_list_help(struct mp_log *log, const m_option_t *opt,
 const struct m_sub_options dec_wrapper_conf = {
     .opts = (const struct m_option[]){
         {"correct-pts", OPT_BOOL(correct_pts)},
-        {"fps", OPT_DOUBLE(force_fps), M_RANGE(0, DBL_MAX)},
+        {"force-container-fps", OPT_DOUBLE(force_fps), M_RANGE(0, DBL_MAX)},
```

Comment:
Maybe `--override-container-fps` to be in parity with `--override-display-fps`

Reply:
Ah yes, consistency is good.

Reply:
Maybe a few other minor adjustments too (`force_fps` -> `override_fps`).